modeler.js

```diff
@@ -0,0 +1,77 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+const authorizationConfiguration = {
+  clientId: process.env.MODELER_CLIENT_ID,
+  clientSecret: process.env.MODELER_CLIENT_SECRET,
+  audience: process.env.MODELER_AUDIENCE
+};
+
+// An action that retrieves a file.
+async function retrieveFile([fileId]) {
+    const accessToken = await getAccessToken(authorizationConfiguration);
+
+    const ModelerBaseUrl = process.env.MODELER_BASE_URL;
```

Comment:
I wasn't sure, do we still need a cluster ID for this one?

Reply:
No, we don't need a cluster ID for any modeler API calls!